cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -853,6 +854,124 @@
     </description>
   </property>
 
+  <property>
+    <name>kafka.bind.address</name>
+    <value>0.0.0.0</value>
+    <description>
+      DEPRECATED: CDAP Kafka service bind address
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.bind.port</name>
+    <value>9092</value>
+    <description>
+      DEPRECATED: CDAP Kafka service bind port
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.default.replication.factor</name>
+    <value>1</value>
+    <description>
+      DEPRECATED: CDAP Kafka service replication factor; used to replicate Kafka messages across
+      multiple machines to prevent data loss in the event of a hardware
+      failure. The recommended setting is to run at least two CDAP Kafka servers.
+      If you are running two CDAP Kafka servers, set this value to 2; otherwise,
+      set it to the maximum number of tolerated machine failures
+      plus one (assuming you have that number of machines).
```

Comment:
With this one, no need to duplicate the entire description as it is lengthy, and is duplicated in the new version:
```
CDAP Kafka service replication factor (deprecated; replaced with kafka.server.default.replication.factor)